internal/healthchecks/healthchecks_test.go

```diff
@@ -38,15 +35,15 @@ func (c FailureCheck) Name() string {
 	return "Failure Check"
 }
 
-func (c FailureCheck) RunCheck(logger *log.Logger) error {
+func (c FailureCheck) RunCheck(logger *logs.FileLogger) error {
 	return healthchecks.HcFailureErr
 }
 
 func TestCheckFailure(t *testing.T) {
 	wantMessage := "The Health Check encountered an internal error."
 	wantAction := "Submit a support case from Google Cloud console."
 	testCheck := FailureCheck{}
-
+	testLogger := logs.DiscardLogger()
```

Comment:
If this is shared by all unit tests would it make sense to keep it at the top (like the old testLogger) ? Or is there a need to have it inside the unit tests ? I think either way is fine.